src/Orchard.Tests/Orchard.Framework.Tests.csproj

```diff
@@ -459,4 +462,10 @@
   <Target Name="AfterBuild">
     <CallTarget Targets="CopySqlCeBinaries" />
   </Target>
+  <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
```

Comment:
Why is this change necessary?

Reply:
it's not. VS had added those lines when I was updating the test adapters for the tests to work properly with VS 2017. I removed it in the latest push